testbdd/README.md

```diff
@@ -0,0 +1,20 @@
+# BDD tests
+
+Tests in this module are a rewrite and enhancement of the end-to-end tests available in the `/test` directory.
```

Comment:
I think it's worth replacing the `test` directory with the BDD one. Also, do we really need it to be in a new exported go module? Where are we going to use it but here?

Reply:
It makes sense, but I'd like to avoid importing kogito-operator here. I think we agreed on creating a new repo for this and refactoring here and there to reuse the common modules.

Reply:
yes but we are working by steps there :)